sparse/src/KokkosSparse_gmres_handle.hpp

```diff
@@ -95,80 +99,71 @@ class GMRESHandle {
                    typename nnz_row_view_t::device_type,
                    typename nnz_row_view_t::memory_traits>;
 
-  using precond_t = Preconditioner<nnz_scalar_t, Kokkos::LayoutLeft,
-                                   ExecutionSpace, nnz_lno_t>;
-
-  enum Ortho { CGS2, MGS };
-  enum Flag { Conv, NoConv, LOA, NotRun };
+  /**
+   * The orthogonalization type
+   */
+  enum Ortho {
+    CGS2,  // Two iterations of Classical Gram-Schmidt
+    MGS
+  };  // One iteration of Modified Gram-Schmidt
+
+  /**
+   * The result of the run
+   */
+  enum Flag {
+    Conv,    // Converged
+    NoConv,  // Did not converge
+    LOA,     // Solver had loss of accuracy
+    NotRun
```

Comment:
Comment should be on previous line. :)

Reply:
@jennloe , this is why I don't like automatic code formatters! It looks like it mangled my Ortho comments too. :(